docs/src/python/pydantic.md

```diff
@@ -13,7 +13,7 @@ via [pydantic_to_schema()](python.md#lancedb.pydantic.pydantic_to_schema) method
 
 ## Vector Field
 
-LanceDB provides a [`Vector(dim)`](python.md#lancedb.pydantic.Vector) method to define a
+LanceDB provides a [`Vector(dim)`](python.md#utilities) method to define a
```

Comment:
It seems you are changing the link target from `lancedb.pydantic.Vector` to `lancedb.vector`.  Should you change the link text from `Vector(dim)` to`vector(dim)`?

Reply:
let me revert the change. vector(dim) will be removed in the future release